scripts/react-gettext-parser-config.js

```diff
@@ -0,0 +1,2 @@
+exports.output = 'locale/react-ui-components.pot';
+exports.funcArgumentsMap = { __: ['msgid'] };
```

Comment:
ui-classic is also exposing `n__` , should we do the same here?

Reply:
We could, although at the moment we don't use it anywhere in the repo. So I did not add it (yet).